pkg/didmethod/httpbinding/resolver.go

```diff
@@ -86,7 +89,7 @@ func containsDIDDocument(resp *http.Response) bool {
 }
 
 // New creates new DID Resolver
-func New(endpointURL string, opts ...ResolverOpt) (*DIDResolver, error) {
+func New(endpointURL string, acceptDID AcceptDID, opts ...ResolverOpt) (*DIDResolver, error) {
```

Comment:
acceptDID isn’t an option?

Reply:
I thought it's mandatory

Reply:
unless the default is return true :)

Reply:
ok I will change to option and default value will be true

Reply:
done